examples/http-request-interceptor/http_request_interceptor.md

```diff
@@ -1,6 +1,6 @@
 # HTTP service - Request interceptor
 
-The `http:RequestInterceptor` is used to intercept the request and execute some custom logic. A `RequestInterceptor` is a service object with only one resource method, which is executed before dispatching the request to the actual resource in the target service. This resource method can have parameters just like a usual resource method in an `http:Service`. A `RequestInterceptor` can be created from a service class, which includes the `http:RequestInterceptor` service type. Then, this service object can be engaged at the listener level or service level by using the `interceptors` field in the configurations. This field accepts an array of interceptor service objects as an interceptor pipeline, and the interceptors are executed in the order in which they are placed in the pipeline. Use `RequestInterceptors` to execute some common logic such as logging, header manipulation, state publishing, etc., for inbound requests.
+The `http:RequestInterceptor` is used to intercept the request and execute some custom logic. A `RequestInterceptor` is a service object with only one resource method, which is executed before dispatching the request to the actual resource in the target service. This resource method can have parameters just like a usual resource method in an `http:Service`. A `RequestInterceptor` can be created from a service class, which includes the `http:RequestInterceptor` service type. Then, this service object can be engaged at the listener level by using the `interceptors` field in the `http:ListenerConfiguration` or at service level by using the `http:InterceptableService` in the service definition. These accept an interceptor service object or an array of interceptor service objects as an interceptor pipeline, and the interceptors are executed in the order in which they are placed in the pipeline. Use `RequestInterceptors` to execute some common logic such as logging, header manipulation, state publishing, etc., for inbound requests.
```

Comment:
```suggestion
The `http:RequestInterceptor` is used to intercept the request and execute custom logic. A `RequestInterceptor` is a service object with only one resource method, which is executed before dispatching the request to the actual resource in the target service. This resource method can have parameters just like a usual resource method in an `http:Service`. 

A `RequestInterceptor` can be created from a service class, which includes the `http:RequestInterceptor` service type. Then, this service object can be engaged at the listener level by using the `interceptors` field in the `http:ListenerConfiguration` or at the service level by using the `http:InterceptableService` in the service definition. 

These accept an interceptor service object or an array of interceptor service objects as an interceptor pipeline, and the interceptors are executed in the order in which they are placed in the pipeline. Use `RequestInterceptors` to execute common logic such as logging, header manipulation, state publishing, etc., for inbound requests.
```
This paragraph is too big and should be broken down to multiple for improved clarity.